### PR TITLE
Accessibility changes for radio buttons

### PR DESCRIPTION
**Pull request by Santino-Trade**

### CODE REVIEW

File: export_support/core/templates/core/business_details_wizard_step.html [hunk 1]
```diff
@@ -16,37 +16,8 @@ <h1 class="govuk-fieldset__heading">
...
-                    <div>
-                        <a id="{{ form.prefix }}-{{ form.company_type.name }}_errors"></a>
-                        {% for error in form.company_type.errors %}
-                            <span class="govuk-error-message">
-                                <span class="govuk-visually-hidden">Error:</span> {{ error }}
-                            </span>
-                        {% endfor %}
-                    </div>
-                {% endif %}
-                <div class="govuk-radios" data-module="govuk-radios">
```
Comment (Santino-Trade) [inline export_support/core/templates/core/business_details_wizard_step.html:37]: Did these radio buttons do something extra that we couldn't use the same way to create them as question page 3 radio buttons?

Comment (kevincarrogan) [reply export_support/core/templates/core/business_details_wizard_step.html:37]: Yes. These did.

The last radio button should display an additional field when selected so you can't just swap this out for the other template.

The bit that's important is the ` data-aria-controls="conditional-type-of-organisation"` (there may be other important bits but that's the one I spotted straight away).

File: assets/scss/styles/forms.scss [hunk 2]
```diff
@@ -23,3 +23,7 @@
   border-top: 1px solid $govuk-border-colour !important;
   padding-top: govuk-spacing(4) !important;
 }
+
+.non-bold-legend {
+  font-weight: normal;
+}
```
Comment (kevincarrogan) [inline assets/scss/styles/forms.scss:29]: Instead of adding this new class use the class `govuk-!-font-weight-regular` instead.

**Review by kevincarrogan — CHANGES_REQUESTED** [review]

File: export_support/core/templates/core/business_details_wizard_step.html (hunk 1 above)
Comment (Santino-Trade) [reply export_support/core/templates/core/business_details_wizard_step.html:37]: Ah, right, yea. Ok, put it back in.

**Review by kevincarrogan — APPROVED** [review]

**Pull request merged**